frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -30,7 +30,11 @@ const forestClientNumberVerifyErrors = ref([] as Array<string>);
 
 const verifyForestClientNumber = async (forestClientNumbers: string) => {
     forestClientNumberVerifyErrors.value = [];
-    let forestNumbers = forestClientNumbers.split(/[ ,]+/);
+
+    // split by commas and spaces
+    let forestNumbers = forestClientNumbers
+        .split(',')
+        .flatMap((clientNum) => clientNum.trim().split(' '));
```

Comment:
It's good you find a way to avoid using regular expression in this case; but I am not sure I follow why using".flatMap" and split again with (' ')? Wondering, vave you try using just ".map()" with trim()?

Reply:
I think it can be simplified to not use "flatMap", I'll take a look. Thanks!